app/views/user_mailer/ranking.html.haml

```diff
@@ -4,9 +4,8 @@
   Congratulations on completing the MITRE CTF! Your team, #{@user.team.team_name} came ranked #{@rank.ordinalize}.
 - if (@rank.eql? 1) && @team.eligible?
   %p
-    Because you finished on the first-place team in the #{@div.name} Division, you could be eligible for a scholarship.  Please email
-    #{link_to 'ctf@mitre.org', 'mailto:ctf@mitre.org'} with your unofficial
-    college, university, or high school transcript AND an up-to-date resume to verify your eligibility.
+    Because you finished on the first-place team in the #{@div.name} Division, you could be eligible for a scholarship.  Please upload your unofficial
+    college, university, or high school transcript AND an up-to-date resume to your profile on the scoreboard to verify your eligibility.  You can do this by clicking #{link_to 'here', edit_user_registration_url}.
```

Comment:
Recommended change: `college, university, or high school transcript AND an up-to-date resume to your #{link_to 'profile', edit_user_registration_url} on the scoreboard to verify your eligibility.`